src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1596,7 +1596,9 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
             INuGetProjectContext nuGetProjectContext,
             IReadOnlyCollection<SourceRepository> activeSources,
             VersionRange versionRange,
-            CancellationToken token)
+            CancellationToken token,
+            string newMappingID = null,
+            string newMappingSource = null)
```

Comment:
I very strongly discourage public APIs from using parameters with default values. Method overloads are, in my opinion, better in every way.